### DISCUSSION TITLE
Is it possible to use <style> tags in the Svelte component?

### DISCUSSION THREAD

**Discussion by matfantinel** [etherCorps/sveltekit-og#48] [Q&A] [answered]
I know _external_ stylesheets aren't supported (a limitation of Satori itself), but can we use `<style>` tags in the Svelte element? It seems to just be getting ignored when I do that.

**Comment by matfantinel** [comment]
Actually, it seems to break whenever I add any element with a child, unless it has the Tailwind styles.

```html
	<div>
		<p>This throws an error</p>
	</div>

	<div tw="flex">
		<p>This works</p>
	</div>
```

Is using Tailwind just pretty much mandatory?

**Reply by theetherGit** [reply]
I do remember to have inline css but not style tag btw you should look this in [@etherCorps/svelte-h2j](https://github.com/etherCorps/svelte-h2j). This is the library i have implemented to render svelte components. I haven't worked on style tag there.

**Reply by theetherGit** [reply]
new beta version fixes all the issues: https://github.com/etherCorps/sveltekit-og/issues/47#issuecomment-3557259389

**Comment by matfantinel** [comment, marked as answer]
Seems like the issue with that specific example is that, on Satori, all divs with more than one child need to have either display: flex or display: none. I'm guessing the tailwind styles do some of that automatically.

**Reply by theetherGit** [reply]
I'm working on new version and I'll try to fix all of your issues. Please let me know if you have any other issue.

**Reply by theetherGit** [reply]
https://github.com/etherCorps/sveltekit-og/issues/47#issuecomment-3557259389 - New beta test
